### PR TITLE
Flowc html no translate

### PR DESCRIPTION
**Pull request by dmitry-vlasov**

### CODE REVIEW

**Comment by DudkevichD** [comment]
https://trello.com/c/4QaGoBQm/

**Comment by DudkevichD** [comment]
Automatic translation is not working now, which is good!

File: tools/flowc/backends/common.flow [hunk 1]
```diff
@@ -659,8 +660,9 @@ getFcBackendConfigs(config : CompilerConfig) {
 				htmlAdditionalScripts,
 				htmlPredefinedParams,
 				getConfigParameterDef(config.config, "working-dir", ""),
-				if (htmlManifest == "1") "manifest.json" else htmlManifest)
-			) else None();
+				if (htmlManifest == "1") "manifest.json" else htmlManifest,
+				getConfigParameterDef(config.config, "html-translate", "") == "1"
```
Comment (NikitaFil) [inline tools/flowc/backends/common.flow:664]: Maybe it is better to name it "html-no-translate" to avoid confusion?

**Review by NikitaFil — APPROVED** [review]
Generally fine

File: tools/flowc/backends/common.flow (hunk 1 above)
Comment (dmitry-vlasov) [reply tools/flowc/backends/common.flow]: Option is renamed to 'html-no-translate', default value is 1.

**Pull request merged**